src/main/java/fi/metatavu/famifarm/events/SowingEventController.java

```diff
@@ -48,15 +48,30 @@
    * @param potType pot type
    * @param amount amount
    * @param additionalInformation additional information
-   * @param modifier modifier
+   * @param creatorId creator id
    * @return updated sowingEvent
    */
   @SuppressWarnings ("squid:S00107")
   public SowingEvent createSowingEvent(Batch batch, OffsetDateTime startTime, OffsetDateTime endTime, ProductionLine productionLine, Collection<SeedBatch> seedBatches, PotType potType, Integer amount, String additionalInformation, UUID creatorId) {
     SowingEvent sowingEvent = sowingEventDAO.create(UUID.randomUUID(), batch, startTime, endTime, productionLine, potType, amount, 0, additionalInformation, creatorId, creatorId);
+    int scaledDifference = getPotTypeAmount(potType) * amount;
```

Comment:
It would be less error prone if the new total would be calculated from data again instead of adding or removing from existing total